src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -153,7 +153,10 @@ def parse_line_for_strategy_prefix(line, strategy_name):
   return stats
 
 
-def parse_performance_features(log_lines, strategies, arguments):
+def parse_performance_features(log_lines,
+                               strategies,
+                               arguments,
+                               include_strategies=True):
```

Comment:
Can't understand why we need `include_strategies` switch. Can we just rely on the `strategies` being (or not being) `None`/`[]`?

Reply:
Ah, I guess it's a temporary solution to support both old launcher and the new code. Maybe add another TODO to remove the strategies from this function completely? Should be doable once we migrate to the new interface + https://github.com/google/clusterfuzz/issues/802 gets resolved.

Reply:
Yes, this is needed to support both at the moment. will add a TODO.